rpcs3/Emu/RSX/rsx_methods.cpp

```diff
@@ -839,7 +839,7 @@ namespace rsx
 
 				default:
 					// Ignored on RSX
-					method_registers.decode(reg, method_registers.register_previous_value);
+					method_registers.registers[reg] = method_registers.register_previous_value;
```

Comment:
I'd rather have a proper method for this like "store" or "set"
It is sometimes important to hook into it to debug stuff, its easier if it all happens in a single place.